src/control/server/mgmt_client.go

```diff
@@ -233,3 +237,23 @@ func (msc *mgmtSvcClient) Stop(ctx context.Context, destAddr string, req *mgmtpb
 
 	return
 }
+
+// Restart calls function remotely over gRPC on server listening at destAddr.
+//
+// Shipped function issues RestartRanks requests using MgmtSvcClient to
+// restart the designated ranks as configured in persistent superblock.
+func (msc *mgmtSvcClient) Restart(ctx context.Context, destAddr string, req *mgmtpb.RestartRanksReq) (resp *mgmtpb.RestartRanksResp, restartErr error) {
```

Comment:
Is there a plan to update the naming of these methods/protobuf definitions as well?